lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
I can confirm, just having setup SkyLight on a project. All I had to do was add the gem and set the Heroku config variable. However, to enable SkyLight on staging (in a `staging` environment), I had to provide `config.skylight.environments << "staging"` in `config/environments/staging.rb`.